Spokestack/SpeechPipeline.swift

```diff
@@ -53,6 +53,30 @@ import Foundation
         wakewordRecognizerService.delegate = nil
     }
     
+    /// Initializes a new speech pipeline instance with reasonable defaults for configuration, wakeword, and asr recognizers.
+    /// - Parameter speechDelegate: An implementation of `SpeechEventListener`.
+    /// - Parameter pipelineDelegate: An implementation of `PipelineDelegate`.
+    @objc public init(_ speechDelegate: SpeechEventListener,
+                      pipelineDelegate: PipelineDelegate) throws {
+        self.speechConfiguration = SpeechConfiguration()
+        self.speechDelegate = speechDelegate
+        
+        self.speechRecognizerService = SpeechProcessors.appleSpeech.processor
+        /// order is important: set the delegate first so that configuration errors/tracing can be sent back
+        self.speechRecognizerService.delegate = self.speechDelegate
+        self.speechRecognizerService.configuration = SpeechConfiguration()
```

Comment:
You probably don't want to bother creating a second instance of this here, right?